build.sbt

```diff
@@ -0,0 +1,50 @@
+import sbtrelease.ReleaseStateTransformations._
+
+name := "effect-util"
+scalaVersion := "2.12.6"
+
+resolvers += Resolver.sonatypeRepo("releases")
+
+val bintrayReleaseProcess: Seq[ReleaseStep] = Seq(
+  releaseStepTask(test),
+  inquireVersions,
+  setReleaseVersion,
+  releaseStepTask(publish),
+  commitReleaseVersion,
+  setNextVersion,
+  commitNextVersion
+)
+
+val common = Seq(
+  organization := "com.ovoenergy",
+  organizationName := "Ovo Energy",
+  organizationHomepage := Some(url("http://www.ovoenergy.com")),
+  bintrayRepository := "maven",
+  bintrayOrganization := Some("ovotech"),
+  releaseCommitMessage := s"Setting version of ${name.value} to ${version.value} [ci skip]",
+  releaseProcess := bintrayReleaseProcess,
+  scalacOptions ++= Seq(
+    "-unchecked",
+    "-deprecation",
+    "-encoding",
+    "utf8",
+    "-feature",
+    "-Xfatal-warnings",
+    "-language:higherKinds"
+  ),
+  libraryDependencies ++= Seq(
+    compilerPlugin("org.spire-math" %% "kind-projector" % "0.9.7"),
+    "org.typelevel" %% "cats-effect" % sys.env.getOrElse("CATS_EFFECT_VERSION", "0.10.1"),
```

Comment:
[Is `0.10` newer then `0.10.1`?](https://github.com/typelevel/cats-effect/tags) Also, how come that you're checking the sys env value?

Reply:
That release section is pretty confusing, if you click into the releases the dates are different, I guess maybe it is showing when they were last edited? 😕 
 
but 0.10.1 is newer than 0.10